README-tmp.md

```diff
@@ -0,0 +1,374 @@
+
+```bash
+export ACCOUNT_B_ROLE_NAME=${CLUSTER_NAME}-cross-account-aws-efs-csi-operator
+```
+
+3. Create the IAM Trust Policy file:
+
+```bash
+cat <<EOF > $SCRATCH_DIR/AssumeRolePolicyInAccountB.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::${AWS_ACCOUNT_A_ID}:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+EOF
+```
+
+4. Create the IAM Role for the EFS CSI Driver Operator:
+
+```bash
+ACCOUNT_B_ROLE_ARN=$(aws iam create-role \
+  --role-name "${ACCOUNT_B_ROLE_NAME}" \
+  --assume-role-policy-document file://$SCRATCH_DIR/AssumeRolePolicyInAccountB.json \
+  --query "Role.Arn" --output text) \
+&& echo $ACCOUNT_B_ROLE_ARN
+```
+
+5. Create the IAM Policy document:
+
+```bash
+cat << EOF > $SCRATCH_DIR/EfsPolicyInAccountB.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeNetworkInterfaces",
+                "ec2:DescribeSubnets"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "elasticfilesystem:DescribeMountTargets",
+                "elasticfilesystem:DeleteAccessPoint",
+                "elasticfilesystem:ClientMount",
+                "elasticfilesystem:DescribeAccessPoints",
+                "elasticfilesystem:ClientWrite",
+                "elasticfilesystem:ClientRootAccess",
+                "elasticfilesystem:DescribeFileSystems",
+                "elasticfilesystem:CreateAccessPoint",
+                "elasticfilesystem:TagResource"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOF
+```
+
+6. Create the IAM Policy:
+
+```bash
+ACCOUNT_B_POLICY_ARN=$(aws iam create-policy --policy-name "${CLUSTER_NAME}-efs-csi-policy" \
+   --policy-document file://$SCRATCH_DIR/EfsPolicyInAccountB.json \
+   --query 'Policy.Arn' --output text) \
+&& echo ${ACCOUNT_B_POLICY_ARN}
+```
+
+7. Attach the Policy to the Role:
+
+```bash
+aws iam attach-role-policy \
+   --role-name "${ACCOUNT_B_ROLE_NAME}" \
+   --policy-arn "${ACCOUNT_B_POLICY_ARN}"
+```
+
+## 5.9.5.5 Configure AWS Account A IAM Roles and Policies
+
+1. Switch to your Account A profile:
+```bash
+export AWS_DEFAULT_PROFILE=${AWS_ACCOUNT_A}
+```
+
+2. Create the IAM Policy document:
+
+```bash
+cat << EOF > $SCRATCH_DIR/AssumeRoleInlinePolicyPolicyInAccountA.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "sts:AssumeRole",
+      "Resource": "${ACCOUNT_B_ROLE_ARN}"
+    }
+  ]
+}
+EOF
+```
+
+3. In AWS Account A, attach the AWS-managed policy "AmazonElasticFileSystemClientFullAccess" to the OpenShift Container Platform cluster master role:
```

Comment:
Currently tested on OCP standalone and OCP HCP kind clusters.
Note: If running on HCP kind cluster no need of executing this 3rd steps. Since HCP cluster has only worker nodes. 
I have tested with all mentioned steps on HCP kind cluster except this 3rd step and we can provision volume, able to read/write/execute. 
We can mark this doc on both OCP standalone and OCP HCP kind clusters. 
cc: @jsafrane @Phaow

Reply:
So do you test without the step volumeReclaimPolicy work as expected? I mean in hcp clusters delete the pvc then the pv should be deleted to be consistent with the `volumeReclaimPolicy:Delete`. The step is used for driver controller reclaim volumes.

Reply:
Yes, it is up to which nodes the driver controller pod located(hcp clusters the driver controller located at workers), thanks for the doulbe checking!